rskj-core/src/main/java/co/rsk/peg/BridgeStorageAccessor.java

```diff
@@ -0,0 +1,72 @@
+package co.rsk.peg;
+
+import co.rsk.core.RskAddress;
+import org.ethereum.core.Repository;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.PrecompiledContracts;
+
+import java.io.IOException;
+
+public class BridgeStorageAccessor {
```

Comment:
I'm not sure about this solution, if extending this class is the best idea. Let's discuss this a bit